### PR TITLE
Endpoint to update a job listing

### PR DESCRIPTION
**Pull request by theijhay**
​

## Description

​

## Related Issue (Link to issue ticket)
#269 

​

## Motivation and Context

​

## How Has This Been Tested?

​

## Screenshots (if appropriate - Postman, etc):

​

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
      ​

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .env
Comment (joboy-dev) [inline .env:1]: Why are you putting  .env in git.  Remove it now please

File: .gitignore
Comment (joboy-dev) [inline .gitignore:1]: Why did you remove .env from .gitignore

File: api/utils/config.py
Comment (joboy-dev) [inline api/utils/config.py:1]: This is not needed. It i already taken care of

File: api/utils/settings.py
Comment (joboy-dev) [inline api/utils/settings.py:1]: What is going on here. Please do not make me close this PR

File: api/v1/routes/job.py
Comment (joboy-dev) [inline api/v1/routes/job.py:1]: USE SERVICES FOR DATABASE QUERIES

File: tests/v1/test_jobs.py
Comment (joboy-dev) [inline tests/v1/test_jobs.py:1]: MOCK THE DATABASE!!!

**Review by joboy-dev — CHANGES_REQUESTED** [review]
Update your PR

**Comment by theijhay** [comment]
Sure! On it
